### PR TITLE
docs(linux): Update readmes

### PR DESCRIPTION
**Pull request by ermshiperete**
- add use of Dev Containers to sample settings
- update installation requirements in README.md

@keymanapp-test-bot skip

### CODE REVIEW

File: docs/settings/linux/.devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/ubuntu/.devcontainer/base.Dockerfile
+
+# [Choice] Ubuntu version (use ubuntu-22.04 or ubuntu-18.04 on local arm64/Apple Silicon): ubuntu-22.04, ubuntu-20.04, ubuntu-18.04
```
Comment (mcdurdin) [inline docs/settings/linux/.devcontainer/Dockerfile:3]: Is this referring to the `VARIANT` arg? Can you make that explicit?

**Review by mcdurdin — APPROVED** [review]
LGTM

File: docs/settings/linux/.devcontainer/Dockerfile (hunk 1 above)
Comment (ermshiperete) [reply docs/settings/linux/.devcontainer/Dockerfile]: Done

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 16.0.83-alpha](https://keyman.com/downloads/releases/alpha/16.0.83)

**Pull request merged**